test.js

```diff
@@ -187,3 +190,13 @@ test('Handles stderr error', async t => {
 	promise.subprocess.stderr.emit('error', error);
 	t.is(await t.throwsAsync(promise), error);
 });
+
+test('Can run global npm binaries', async t => {
+	const {stdout} = await nanoSpawn('npm', ['--version'], {shell: isWindows});
```

Comment:
See #12.
The way npm global binaries are installed on Windows relies on the presence of a shell. :-/
We can try to make it work on Windows in a separate PR, if this is not too much code.